docs/installation.rst

```diff
@@ -144,52 +107,18 @@ To test if all the required files have been installed, please import STIPS in py
 
     >>> import stips
 
-The following warning message can be ignored if it appears::
-
-    WARNING: stips_data environment variable not found. Falling back on local STIPS data.
-
-
-Installing Using Docker
-#######################
+		    print(stips.__env__report__)
```

Comment:
```suggestion
    >>> print(stips.__env__report__)
```
The added line of Python is missing the prompt and is not on the same indentation level as the existing one.